### PR TITLE
Added  post highlight section 1

### PR DESCRIPTION
**Pull request by srikant96**

### CODE REVIEW

File: blogs/01/styles/post-highlight-section-1.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+section.post-highlight-section-1{
+    background-color: #f8f9fa!important;
```
Comment (amezng) [inline blogs/01/styles/post-highlight-section-1.css:2]: Do not use `!important` flag.

File: blogs/01/styles/post-highlight-section-1.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+div.highlight-section-wrapper{
+    width: 1125px;
+    height:590px;
+    display:flex;
+    justify-content:space-evenly;
+    align-items: center;
+	margin:auto;
+    gap:20px;
```
Comment (amezng) [inline blogs/01/styles/post-highlight-section-1.css:12]: align the margin attribute

**Review by amezng — CHANGES_REQUESTED** [review]
MTC

**Comment by amezng** [comment]
Add the screenshot of the section

File: blogs/01/styles/post-highlight-section-1.css (hunk 2 above)
Comment (srikant96) [reply blogs/01/styles/post-highlight-section-1.css:12]: Done

File: blogs/01/styles/post-highlight-section-1.css (hunk 1 above)
Comment (srikant96) [reply blogs/01/styles/post-highlight-section-1.css]: Done

**Review by amezng — APPROVED** [review]
LGTM

**Pull request merged**